concourse/pipelines/rhui-release.jsonnet

```
@@ -0,0 +1,142 @@
+local project = 'google.com:rhel-infra';
+
+local gatejob = {
+  local job = self,
+
+  name: error 'must set name on gatejob',
+  trigger:: true,
+  passed:: [],
+  plan: [
+    {
+      get: imageResource,
+      trigger: job.trigger,
+      passed: job.passed,
+    }
+    for imageResource in ['cds-image', 'rhua-image']
+  ],
+};
+
+local gcloudmigtask = {
+  local task = self,
+
+  node:: error 'must set node on gcloudmigtask',
+  stage:: 'prod',
+  region:: error 'must set region on gcloudmigtask',
+  action:: error 'must set action on gcloudmigtask',
+
+  task: error 'must set task',
+  config: {
+    image_resource: {
+      source: {
+        repository: 'google/cloud-sdk',
+        tag: 'alpine',
+      },
+      type: 'registry-image',
+    },
+    platform: 'linux',
+    run: {
+      args: [
+        'compute',
+        'instance-groups',
+        'managed',
+      ] + task.action + [
+        '--quiet',
```

`+` will only work if `task.action` is also an array. suggest:

```suggestion
        'compute',
        'instance-groups',
        'managed',
        task.action,
        '--quiet',
```

---

`task.action` _is_ an array :)

---

ok, i see. i thought task.action would always be a single word, but i see it is almost always multiple words.